src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/SliderExtensions.cs

```diff
@@ -7,24 +7,18 @@ namespace Xamarin.Platform
 {
 	public static class SliderExtensions
 	{
-		public static void UpdateMinimum(this SeekBar seekBar, ISlider slider)
-		{
-			if (NativeVersion.Supports(NativeApis.SeekBarSetMin))
-			{
-				seekBar.Min = (int)slider.Minimum;
-			}
-		}
+		static double Min;
```

Comment:
I don't think these can be in here because if you have different sliders then these will just keep changing. Max and Min are part of ISlider so you don't need to store these here for UpdateValue and/or you can just have a method that takes in a max/min parameter

Reply:
Right, updated.